src/server/sourceSelect.js

```diff
@@ -49,7 +49,11 @@ const constructPathToGet = (source, providedUrl) => {
   let datasetFields; // this _does not_ take into account the 2nd tree
   let treeName;
 
-  const parts = providedUrl.replace(/^\//, '').replace(/\/$/, '').split("/");
+  const parts = providedUrl
+    .replace(/^\//, '')
+    .replace(/\/$/, '')
+    .replace("live/", "")
```

Comment:
This should probably be anchored to the start of the string, otherwise no dataset can include a path segment ending in `live`.

Reply:
yup, done